### ISSUE TITLE
unicode inspection does not strip `unpack-XXXX` directories anymore

### ISSUE THREAD

**Issue by jchecahi** [rpminspect/rpminspect#1440]
**Describe the bug**
This is a regression of #1167 and it looks like it was introduced in #1437. This means that some exceptions (maybe all?) set in rpminspect-data-* packages (like [these](https://github.com/rpminspect/rpminspect-data-fedora/blob/main/security/fc41)) no longer have effect and will cause false positives.

This was reproduced with latest copr build rpminspect-2.1-0.1.202410161454gitf11af6a9.fc41.x86_64

**To Reproduce**
Using annobin as example as it's quick to run on:
```
$ rpminspect-fedora -v -T unicode -a src annobin-12.69-1.fc41
[...]
unicode:
--------
1) A forbidden code point, 0x202C, was found in the unpack-zcU1jO/annobin-12.69/tests/trick-hello.s source file on
line 12 at column 14.  This source file is used by annobin.spec.

Result: BAD
Waiver Authorization: Security

Suggested Remedy:  6gUV

2) A forbidden code point, 0x202E, was found in the unpack-zcU1jO/annobin-12.69/tests/trick-hello.s source file on
line 12 at column 10.  This source file is used by annobin.spec.

Result: BAD
Waiver Authorization: Security

Suggested Remedy:  6gUV
[ ... ]
```
The same file is reported more times. The code point should have been ignored per [this](https://github.com/rpminspect/rpminspect-data-fedora/blob/main/security/fc41#L88) rpminspect-data-redhat config but the unstripped `unpack-XXXXXX/` makes the rule to *not* match.

**Expected behavior**
The unicode exception should report `OK`.

**Environment:**
Fedora 41 (x86_64)
